prep-javascript-functions/main.js

```diff
@@ -0,0 +1,53 @@
+function addTwoNumbers(num1, num2) {
+  return num1 + num2;
+}
+var addTwoNumbersResult = addTwoNumbers(2, 2);
+console.log('addTwoNumbers Exercise:', addTwoNumbersResult);
+
+function convertHoursToMinutes(hours) {
+  return hours * 60;
+}
+var convertHoursToMinutesResult = convertHoursToMinutes(2);
+console.log('convertHoursToMinutes Exercise:', convertHoursToMinutesResult);
+
+function getGreeting(name) {
+  return 'Hello World!';
+}
+var getGreetingResult = getGreeting('World');
+console.log('getGreeting Exercise:', getGreetingResult);
+
+function addAndMultiplyBy5(num1, num2) {
+  return (num1 + num2) * 5;
+}
+var addAndMultiplyBy5Result = addAndMultiplyBy5(10, 5);
+console.log('addAndMultiplyBy5 Exercise:', addAndMultiplyBy5Result);
+
+function multiplyAndDivideBy5(num1, num2) {
+  return (num1 * num2) / 5;
+}
+var multiplyAndDivideBy5Result = multiplyAndDivideBy5(35, 10);
+console.log('multiplyAndDivideBy5 Exercise:', multiplyAndDivideBy5Result);
+
+function subtractTwoNumbers(num1, num2) {
+  return (num1 - num2);
+}
+var subtractTwoNumbersResult = subtractTwoNumbers(22, 7);
+console.log('subtractTwoNumbers Exercise:', subtractTwoNumbersResult);
+
+function getCircleCircumference(radius) {
+  return (2 * radius * 3.14);
+}
+var getCircleCircumferenceResult = getCircleCircumference(5);
+console.log('getCircleCircumference Exercise:', getCircleCircumferenceResult);
+
+function getFullName(firstName, lastName) {
+  return 'Juan' + ' Ramirez';
```

Comment:
Hi Patricia, for this function we want to use the parameters in our concatenation, so that it will take any first and last name and combine them into a full name, not just Juan Ramirez